src/Utils.php

```diff
@@ -736,7 +736,7 @@ function wf_civicrm_api3_contribution_transact($params) {
     $order = civicrm_api3('Order', 'create', $params);
     try {
       $params['amount'] = $params['total_amount'];
-      $params['contribution_id'] = $order['id'];
+      $params['contribution_id'] = $params['contributionID'] = $order['id'];
```

Comment:
Why is this change needed?

Reply:
Since CiviCRM 5.39 this should not be required because of https://github.com/civicrm/civicrm-core/commit/5d8d2f382f3f6662c1e789c76a3d5033ce01c9d6 and core authnet should automatically convert `contribution_id` to `contributionID` internally.

Reply:
Yeah, I'm not entirely sure but this is where it's happening - https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/Payment/AuthorizeNet.php#L170.

Reply:
ok, so the fix should be in CRM/Core/Payment/AuthorizeNet.php and not here. Change the $params['contributionID'] to $propertyBag['contributionID']